src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
```

Comment:
the height and width of an image should be set directly in HTML if they are static, otherwise, you can set the width to 100%. It is due to the browser knowing how much space an image requires when it builds the page